.gitlab-ci.yml

```diff
@@ -112,6 +112,30 @@ build linux arm64:
     - job: "build synapse arm64"
       optional: true
 
+build linux meson wraps:
+  stage: build
+  image: alpine:latest
+  tags: [docker]
+  needs:
+    - job: "build synapse amd64"
+      optional: true
+  variables:
+    M_ARCH: x86_64
+    COVERAGE: "ON"
+  services: !reference [.build-linux, services]
+  before_script:
+    - echo 'https://dl-cdn.alpinelinux.org/alpine/edge/testing' >> /etc/apk/repositories
+    - apk update && apk add meson git g++ cmake pkgconf openssl openssl-dev make
```

Comment:
Do you need openssl and openssl-dev if you're using the meson wrap for them?

Reply:
Yes, the wrap doesn't work for the cmake projects like curl. So basically it is only useful on platforms, where curl doesn't need openssl.